### PR TITLE
feat: feedback similar software

### PR DESCRIPTION
**Pull request by ZekeyWu**
#543

### CODE REVIEW

File: packages/shared-components/components/software-details/SoftwareDetails.vue [hunk 1]
```diff
@@ -1096,9 +1156,57 @@ onBeforeUnmount(() => {
...
+          @close="cancelFeedback"
+        >
+          <el-form
+            ref="formInstance"
+            :model="feedbackInfo"
+            :rules="formRules"
+            label-position="right"
+            label-width="auto"
+          >
+            <el-form-item label="上游社区地址" prop="repoUrl">
```
Comment (ygj6) [inline packages/shared-components/components/software-details/SoftwareDetails.vue]: 社区源码仓地址

File: packages/shared-components/components/software-details/SoftwareDetails.vue [hunk 2]
```diff
@@ -1096,9 +1156,57 @@ onBeforeUnmount(() => {
...
+              <el-input
+                v-model="feedbackInfo.repoUrl"
+                type="textarea"
+                :row="3"
+                placeholder="请输入上游社区地址"
```
Comment (ygj6) [inline packages/shared-components/components/software-details/SoftwareDetails.vue]: https://github.com/owner-name/repo-name

File: packages/shared-components/components/software-details/SoftwareDetails.vue (hunk 1 above)
Comment (ZekeyWu) [reply packages/shared-components/components/software-details/SoftwareDetails.vue]: Code modified

File: packages/shared-components/components/software-details/SoftwareDetails.vue (hunk 2 above)
Comment (ZekeyWu) [reply packages/shared-components/components/software-details/SoftwareDetails.vue]: Code modified

**Pull request merged**